### PR TITLE
Fixes #1184 : Adds auto populating of pricing on fresh install

### LINKED ISSUE #1184: Auto populate pricing on fresh install
**I'm submitting a ...**
- [ ] bug report
- [X] feature request

Adding feature to auto populate the pricing table on the fresh install of badgeyay. 
When the app will be installed, then the initial pricing will be set to $0.00 by default.

**I am working on this issue**

### PR DESCRIPTION
**Pull request by gabru-md**
Fixes #1184 

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Automatically populates pricing model on fresh install

Cheers!

### CODE REVIEW

File: api/config/config.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 DEBUG = True
 
 # Environment for working, can be LOCAL/PROD
-ENV = 'PROD'
+ENV = 'LOCAL'
```
Comment (ParthS007) [inline api/config/config.py]: Keep it `PROD`

File: api/models/utils.py [hunk 2]
```diff
@@ -17,3 +17,8 @@ def save_to_db(self):
             db.session.rollback()
             db.session.flush()
             print(e)
+
+
+def insert_initial_values():
+    util = Utilities(pricing=0.00)
```
Comment (yashLadha) [inline api/models/utils.py]: Won't this will insert new entry every time server restarts and gets first request.

Comment (yashLadha) [reply api/models/utils.py]: Instead of autopopulating id, check for already present id and if not present them the same in the DB

Comment (gabru-md) [reply api/models/utils.py]: Done @yashLadha

**Comment by gabru-md** [comment]
@yashLadha merge?

**Pull request merged**